core/src/main/java/com/scalar/db/storage/jdbc/JdbcAdmin.java

```diff
@@ -709,6 +714,9 @@ private void updateTableMetadata(
   }
 
   static void execute(Connection connection, String sql) throws SQLException {
+    if (Strings.isNullOrEmpty(sql)) {
+      return;
```

Comment:
See https://github.com/scalar-labs/scalardb/blob/4a0b10d44547af4bf4e80ad333da74040cda6357/core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineSqlite.java#L186-L190

`RdbEngine*.*Sql()` methods sometimes return null instead of a SQL string.
To avoid frequent null (or empty string) checking after calling `RdbEngine*.*Sql()` methods, I put such early-return here.

I don't think this is the best solution, so if you have better one, I may follow it.